mesonbuild/mtest.py

```diff
@@ -1528,6 +1528,7 @@ def get_test_setup(self, test: T.Optional[TestSerialisation]) -> build.TestSetup
             return self.build_data.test_setups[self.options.setup]
         else:
             full_name = test.project_name + ":" + self.options.setup
+            print(self.build_data.test_setups)
```

Comment:
Is this a leftover debug print or intentional?

Reply:
I think debug print, I've removed it.